### ISSUE TITLE
ModuleNotFoundError with newest versions of pynitrokey and spsdk

### ISSUE THREAD

**Issue by samweisgamdschie** [Nitrokey/pynitrokey#184]
Hello, I get this error when trying to run `nitropy`.

Python version: 3.8.12
OS: OpenSUSE Tumbleweed
pynitrokey==0.4.13
spsdk==1.6.0

```python
$ nitropy 
Traceback (most recent call last):
  File "/home/a/python/nitrokey/bin/nitropy", line 5, in <module>
    from pynitrokey.cli import nitropy
  File "/home/a/python/nitrokey/lib/python3.8/site-packages/pynitrokey/cli/__init__.py", line 20, in <module>
    from pynitrokey.cli.nk3 import nk3
  File "/home/a/python/nitrokey/lib/python3.8/site-packages/pynitrokey/cli/nk3/__init__.py", line 21, in <module>
    from pynitrokey.nk3 import list as list_nk3
  File "/home/a/python/nitrokey/lib/python3.8/site-packages/pynitrokey/nk3/__init__.py", line 13, in <module>
    from .bootloader import Nitrokey3Bootloader
  File "/home/a/python/nitrokey/lib/python3.8/site-packages/pynitrokey/nk3/bootloader.py", line 18, in <module>
    from spsdk.sbfile.images import BootImageV21
ModuleNotFoundError: No module named 'spsdk.sbfile.images'
```

When changing bootloader.py line 18 to:

`from spsdk.sbfile.sb2.images import BootImageV21`

it works:

```bash
$ nitropy 
Usage: nitropy [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  fido2    Interact with Nitrokey FIDO2 keys, see subcommands.
  ls       List Nitrokey keys (in firmware or bootloader mode)
  nethsm   Interact with NetHSM, see subcommands.
  nk3      Interact with Nitrokey 3, see subcommands.
  pro      Interact with Nitrokey Pro keys, see subcommands.
  start    Interact with 'Nitrokey Start' keys, see subcommands.
  storage  (experimental) 'Nitrokey Storage' keys, see subcommands.
  version  Version of pynitrokey library and tool.
```

**Comment by szszszsz** [comment]
Hi!
Indeed, we need to make a new release yet adjusted to spsdk 1.6.0. I am just working on it. For a quick workaround you can downgrade spsdk to 1.5.0, like:
```
python3 -m pip install -U spsdk==1.5.0
```

Details:
- https://github.com/Nitrokey/pynitrokey/pull/180#issuecomment-1031209339

**Comment by samweisgamdschie** [comment]
OK thanks. Wow, github didnt show me the PR.

**Comment by szszszsz** [comment]
New release `0.4.15` is already available on PIP. To update:
```
python3 -m pip install -U --user pynitrokey
```

Please reopen in case it would not work for you. Closing as done.

**Comment by samweisgamdschie** [comment]
Can confirm it is fixed with `0.4.15`. Thanks for the pretty fast reply!

**Comment by szszszsz** [comment]
Great! Thank you for retesting!
